projects/canopy/src/lib/side-nav/side-nav-bar/side-nav-bar.component.ts

```diff
@@ -0,0 +1,25 @@
+import {
+  ChangeDetectionStrategy,
+  Component,
+  HostBinding,
+  Input,
+  ViewEncapsulation,
+} from '@angular/core';
+
+@Component({
+  selector: 'lg-side-nav-bar',
+  templateUrl: './side-nav-bar.component.html',
+  styleUrls: ['./side-nav-bar.component.scss'],
+  encapsulation: ViewEncapsulation.None,
+  changeDetection: ChangeDetectionStrategy.OnPush,
+})
+export class LgSideNavBarComponent {
+  @Input() label = 'Nav';
+
+  @HostBinding('class.lg-side-nav-bar') class = true;
+  @HostBinding('attr.role') ariaRole = 'tablist';
```

Comment:
I think the aria role should be `navigation`, not tablist, as this will just be a sub menu of links, right?

Reply:
Yep, that's correct.